queries/styled/highlights.scm

```diff
@@ -0,0 +1,4 @@
+; inherits: css
+
+(plain_value) @variable
+(js_expression) @variable
```

Comment:
Shouldn't this be highlighted as `@none` and injected with javascript?

Reply:
I ended up removing them. At one point it was necessary to preventing CSS keywords such as `color: red` from coloring the `red` as a string. I'm not seeing that anymore if I remove them though, so going to remove for now.

Reply:
I still think `(js_expression)` might need to be injected with javascript.

Reply:
From my testing it's not really required, highlighting in the template string works fine.

<img width="327" alt="image" src="https://github.com/nvim-treesitter/nvim-treesitter/assets/25914066/052c2b22-1abc-4314-a701-9f30fa0b8eb6">

Even nested expressions work just fine.

<img width="486" alt="image" src="https://github.com/nvim-treesitter/nvim-treesitter/assets/25914066/a40a5861-e605-4e22-82e6-13e15f52128a">